src/Microsoft.DocAsCode.Common/Path/RelativePath.cs

```diff
@@ -40,6 +41,10 @@ private RelativePath(bool isFromWorkingFolder, int parentDirectoryCount, string[
         #endregion
 
         #region Public Members
+        public static RelativePath FromUrl(string path)
+        {
+            return (RelativePath)Uri.UnescapeDataString(path);
```

Comment:
I don't think this method needs to be inside `RelativePath`, the one calling `RelativePath` knows how to pre-process `path`, `FromUrl` sounds obscure to me what the input it is expecting.